src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/Window.cs

```diff
@@ -257,21 +217,10 @@ public bool IsOwner
         {
             get
             {
-                return InteropAndHelpers.GetWindow(Hwnd, InteropAndHelpers.GetWindowCmd.GW_OWNER) != null;
+                return NativeMethods.GetWindow(Hwnd, NativeMethods.GetWindowCmd.GW_OWNER) != null;
             }
         }
 
-        /// <summary>
-        /// Gets a value indicating whether is the window cloaked. To detect UWP apps in background or win32 apps running in another virtual desktop
-        /// </summary>
-        public bool IsWindowCloaked()
```

Comment:
Does the ww plugin not detect background apps that are running?

Reply:
It will detect them.  It maybe filtered them out at one point before being integrated into Run? I removed this code as it was unused and I would need to make FX cop fixes to p/invokes that I couldn't test.